lib/bridge.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Bridge
+ * @constructor
+ */
+var Bridge = function (client) {
+	/**
+	 * Create a new bridge
+	 * @param {Object} params Parameters for creating a bridge
+	 * @param {Boolean} params.bridgeAudio Enable/Disable two way audio path (default = true).
+	 * @param {Array<String>} params.callIds The list of call ids in the bridge. If the list of call ids
+	 * is not provided the bridge is logically created and it can be used to place calls later.
+	 * @param {Function} [callback] Callback with the newly created bridge
+	 * @return {BridgeResponse} A promise for the newly created bridge
+	 */
+	this.create = function (params, callback) {
+		return client.makeRequest({
+			path   : "bridges",
+			method : "POST",
+			body   : params
+		})
+		.then(function (response) {
+			var bridge = params;
+			var location = response.headers.location;
+			bridge.id = location.substring(location.lastIndexOf("/") + 1);
+			return bridge;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Gets information about a bridge.
+	 * @param  {String} bridgeId The ID of the bridge to get
+	 * @param  {Function} callback A callback with the call information
+	 * @return {Promise} A promise for the call information
+	 */
+	this.get = function (bridgeId, callback) {
+		return client.makeRequest({
+			path   : "bridges/" + bridgeId,
+			method : "GET"
+		})
+		.then(function (response) {
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Gets a list of bridges.
+	 * @param {Object} params Query parameters for listing bridges
+	 * @param {Number} [params.page=0] Used for pagination to indicate the page requested for querying
```

Comment:
We don't want to document the page parameter as we're encouraging people to use the links in the returned header. Eventually we'll move from page & size to a sort key. This way we don't have to update the SDK